.github/workflows/build.yml

```diff
@@ -16,7 +16,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Format Rust code
+      - name: Install nightly rustfmt
+        run: |
+          rustup default nightly
```

Comment:
I actually find it nice to have a unified default for a job — in this case `nightly`. The alternative requires me to decide on a line by line basis (and has the potential of me forgetting to add `+foo` on a line).

But since you both seem to suggest this approach, I updated the PR to match. I'll eventually ditch this for something like `dprint` instead to get unified formatting of everything.